pubsub/rabbitpubsub/rabbit.go

```diff
@@ -533,11 +557,12 @@ type subscription struct {
 
 var nextConsumer int64 // atomic
 
-func newSubscription(conn amqpConnection, name string) *subscription {
+func newSubscription(conn amqpConnection, name string, opts *SubscriptionOptions) *subscription {
 	return &subscription{
```

Comment:
Let's make sure `opts` is non-nil here:

```
if opts == nil {
		opts = &SubscriptionOptions{}
	}
```

Reply:
done in c47bd5488d052b276f080f8c4130edcbc2aa361d